tests/test_widgets.py

```diff
@@ -8,22 +9,33 @@
 )
 
 
+class PhonePrefixSelectTest(SimpleTestCase):
+    def test_without_babel(self):
+        import phonenumber_field
+
+        babel_module = phonenumber_field.widgets.babel
+        phonenumber_field.widgets.babel = None
```

Comment:
Please make sure the cleanup is always executed, e.g. with https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup.

Reply:
Good point. I think using `setUp` and `tearDown` should be enough. The only difference between `tearDown` and registering a function with `addCleanup` is that if won't be fired if `setUp` fails whilst cleanup method is fired always.